### PR TITLE
ar/feature/kept-toggle

### PR DESCRIPTION
**Pull request by srslie**
### Description
- adds toggle kept functionality

#### Closes Issues 
#41, #46

## Proposed Changes
- adds toggle function to display users products they've kept
- changes button styling, h3 to h4 for consistency
- misc inline styling
  
## Considerations
- we have it in the commented out 'map' through so when we get the actual array of userproducts we can just comment that in, delete the examples hard-coded here, and go to town.
- the api to fetch those user products may go in a 'componentDidMount' here

### CODE REVIEW

File: code/web/src/modules/user/MyAccount.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ class MyAccount extends PureComponent {
   renderDescriptionDefaultView = () => {
     return (
       <div style={{ display: 'flex', alignItems: 'center' }}>
-        <p style={{ color: black, maxHeight: '40%', overflow: 'scroll' }}>{this.state.description}</p>
```
Comment (tashiad) [inline code/web/src/modules/user/MyAccount.js:75]: thanks for fixing this!

File: code/web/src/modules/user/MyProducts.js [hunk 2]
```diff
@@ -46,12 +52,12 @@ class MyProducts extends PureComponent {
...
         <div>
-          <H3>Delivery History</H3>
+          <H4 style={{textAlign: 'center', margin: '1em'}}>Delivery History</H4>
           {/* Top actions bar */}
           <Grid alignCenter={true} style={{ padding: '1em' }}>
             <GridCell style={{ textAlign: 'right' }}>
               <Button theme="primary" style={{ marginTop: '1em' }} onClick={this.toggleKept} >
-                Toggle Kept Products
+                {this.state.viewAll ? 'Show Kept' : 'Show All'}
               </Button>
```
Comment (tashiad) [inline code/web/src/modules/user/MyProducts.js:61]: Great addition

**Review by tashiad — APPROVED** [review]

File: code/web/src/modules/user/MyProducts.js [hunk 3]
```diff
@@ -28,10 +28,16 @@ class MyProducts extends PureComponent {
   }
 
   toggleKept = () => {
-    // event.preventDefault()
     this.setState({ viewAll: !this.state.viewAll })
   }
 
+  displayKept = (keptStatus) => {
```
Comment (gaj23) [inline code/web/src/modules/user/MyProducts.js:34]: Nice, thank you. I'll be able to work and finish more of the cypress testing.

**Review by gaj23 — APPROVED** [review]
This sets up the Cypress testing to be completed.

File: code/api/src/config/database.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "development": {
-    "username": "tashiadavis",
+    "username": "alice",
```
Comment (catherinemdean15) [inline code/api/src/config/database.json:3]: Hopefully we can figure something out so we don't keep having to do this every PR! 👍

**Review by catherinemdean15 — APPROVED** [review]

**Review by rachelbuchta — APPROVED** [review]
Great work on this! Love this feature.

**Pull request merged**